ci/jobs/deploy_commit_hooks.yml.j2

```diff
@@ -3,16 +3,14 @@ jobs:
     max_in_flight: 1
     plan:
       - in_parallel:
-          - get: dataworks-github-config
+          - get: dataworks-github-config-webhooks
             trigger: true
-            passed:
```

Comment:
you might still want this to pass through master to cover a scenario where the githook is changing at the same time as a repo is being created

Reply:
This is why I'd prefer an external asynchronous task to do this, instead of a pipeline.  But for now I'd rather avoid this job running as much as possible.   

For the above scenario to occur, two PR's would have to go through at the same time, which can't happen?  So you would either get a new repo, followed by new githooks, which will update the new repo.  Or you will get new githooks, followed by a new repo, which will have the new githooks from the template.